ocaml/testsuite/tests/typing-layouts-float64/unboxed_floats.ml

```diff
@@ -1,22 +1,23 @@
 (* TEST
+   * flambda2
```

Comment:
Why was this change necessary? I remember hearing that all the backends should be working.

Reply:
That was true before literals, but I assume this is necessary because the literals themselves have only been implemented for flambda2?  I am sad about that, but not sad enough to slow this down.

Reply:
Yes, unboxed literals are currently only supported on flambda2.

The first implementation of the feature was compatible with all middle/backends when it compiles to calling unbox on a boxed constant. But when #2062 and #2071 changed the implementation to compile directly to an unboxed constant, closure and flambda1 support were dropped.